src/routes/noplace/index.js

```diff
@@ -0,0 +1,23 @@
+import Heading from '../../components/Heading';
+import BackCar from '../../components/Svg/BackCar';
+import GrayCar from '../../components/Svg/GrayCar';
+import style from './style.scss';
+
+const NoPlaceLeft = () => (
+  <div className={style.contentWrapper}>
+    <div className={style.splashScreen}>
+      <BackCar className={style.backCar} />
+      <GrayCar className={style.grayCar} />
+    </div>
+    <div>
+      <Heading color='Blue'>
+        Dziś nie ma wolnych miejsc
+      </Heading>
+      <p className={style.graySubheader}>
```

Comment:
You can use the `hint` component with a modifier.

Reply:
Yea, I will after merge with 'your-place-view' branch. Already refactored it so it's ready to use it here :)